task_034.py

```diff
@@ -0,0 +1,29 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает,
+#  Вам стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг от друга пробелами. 
+# Стихотворение  Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+#     **Вывод:** Парам пам-пам  
+
+def is_rhyme(phrase):
+    list_v =[]
+    List_phrase = phrase.split()
+    for i in List_phrase:
+        s = 0
+        k = [s+1 for letter in i if letter in "уеыаоэяию"]
+        list_v.append(sum(k))
+    return list_v
+
+def validate(list_v):
+    flag = True
+    for A in range(len(list_v)):
+        max_v = list_v[0] 
+        if list_v[A] != max_v: 
+            flag = False
+    if flag == False: return print("Пам парам")
+    else: return print("Парам пам-пам")  
```

Comment:
Во-первых, запись блока кода в одну строку с условием условия хоть и принята, но не рекомендуется. Во-вторых, у вас печатается `None` в конце, потому что после  `return` у вас по сути и будет `None`, так как `None` – это то, что возвращает функции `print` (тоже самое будет если сделать `print(print())`). Вывод в консоль и возврат из функции – не одно и тоже. Различайте эти 2 понятия.
```suggestion
    if flag == False:
        return "Пам парам"
    else:
        return "Парам пам-пам"
```